src/components/chat-input.scss

```diff
@@ -0,0 +1,45 @@
+@use "vars" as *;
+
+.chat-input {
+  margin-top: 10px;
+
+  fieldset {
+    border: none;
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0;
+    padding: 0;
+  }
+
+  textarea {
+    border: solid 1px $medium-gray;
+    border-radius: 6px;
+    flex: 1 0 calc(100% - 22px); // full width of container minus 10px padding and 1px border on either side
+    height: 75px;
+    padding: 8px 10px;
+  }
+
+  .buttons-container {
+    align-items: center;
+    display: flex;
+    flex-wrap: nowrap;
+    justify-content: space-between;
+    margin: 10px 0;
+    padding: 0;
+    width: 100%;
+
+    button {
+      display: block;
+      flex: 0 0 auto;
+      margin: 0;
+      width: auto;
+
+      &.send {
+        order: 2;
+      }
```

Comment:
I'm sure UI will be handled later, but it's odd that the 'send' button is different styling from the other 3 buttons (grey & square rather than blue & rounded).

Reply:
True. It will be changed later, but since it's easy, I've updated the CSS so it matches the others.